Makefile

```diff
@@ -35,7 +35,7 @@ train:
 
 	python src/train.py \
 		--config-path "config/train_config.yaml" \
-		--train-path "${path}/train.parquet" \
+		--train-path "${path}/train_enhanced.parquet" \
```

Comment:
Do we want the enhanced dataset to be the default going forward? Or could we replace train.parquet if we decide to use the enhanced as the new default?

Reply:
No this should just refer to train_parquet. If the addition of synthetic pubs proves much better we should set up the option in the config and add them in in `preprocess.py`.